python_apps/airtime-celery/README.md

```diff
@@ -0,0 +1,63 @@
+# airtime-celery
+
+airtime-celery is a [Celery](http://www.celeryproject.org/) daemon for handling backend tasks asynchronously.
+Communication and the Celery results backend are both handled with amqp (RabbitMQ).
+
+# Installation
+
+```sh
+sudo python setup.py install
+```
+
+Each instance of airtime-celery has its own worker, and multiple instances can be run in parallel.
+[Celery is thread-safe](http://celery.readthedocs.org/en/latest/userguide/application.html), so this parallelization won't cause conflicts.
+
+# Usage
+
+This program must be run with sudo:
+
+```sh
+sudo service airtime-celery {start | stop | restart | graceful | kill | dryrun | create-paths}
```

Comment:
Can be replaced with systemctl calls, I'm not sure how we handle the non-standard commands though

Reply:
Will do in another PR, this was mainly converting a .rst file to markdown.